packages/ui/src/containers/Passwordless/EmailPasswordless.tsx

```diff
@@ -37,7 +40,16 @@ const EmailPasswordless = ({ type, className }: Props) => {
 
   const errorHandlers: ErrorHandlers = useMemo(
     () => ({
-      'user.email_not_exists': () => {
+      'user.email_not_exists': (error) => {
+        const socialToBind = getSearchParameters(location.search, SearchParameters.bindWithSocial);
+
+        // Directly throw error if user is trying to bind with social
```

Comment:
nit: This is not `throw` error. Might be better to say `popup` or `show` error

Reply:
updated thx